seasmon_xr/ops/autocorr.py

```diff
@@ -215,7 +215,7 @@ def autocorr_tyx(tyx, nodata=None):
         Lag-1 autocorrelation array Y,X order
     """
     _, nr, nc = tyx.shape
-    z = zeros((nr, nc), dtype="float32")
+    z = zeros((nr, nc), dtype=float32)
```

Comment:
is there any specific reason you use the `numba` type here? Not that I think it makes a big difference, just curious.

Reply:
Using the basic types, it gives me this error attached. I used the numba type to resolve the conflict. I'm not sure I have this only on my side though
![type_error](https://user-images.githubusercontent.com/60544544/182101405-eb1b660c-de75-47a3-828d-1e0a6177940b.JPG)

Reply:
hmmm, maybe a windows thing. If it works for you, keep it.